games-emulation/retroarch/retroarch-1.7.1.recipe

```diff
@@ -63,7 +61,6 @@ INSTALL()
 	mkdir -p $includeDir
 
 	cp retroarch $appsDir/RetroArch
```

Comment:
Any reasons not to use `make install`?

Reply:
That's probably because the recipe we already have in the repo does not use `make install`.
I don't know whether or not this is OK, but we will investigate this.
BTW, I noticed that @gouchi is one of the contributors of https://github.com/libretro/RetroArch :)